Frontend/Pages/Projects/GeneralInformation/Index.cshtml.cs

```diff
@@ -0,0 +1,54 @@
+using System.Threading.Tasks;
+using Frontend.ExtensionMethods;
+using Frontend.Models;
+using Frontend.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Frontend.Pages.Projects.GeneralInformation
+{
+    public class Index : CommonPageModel
+    {
+        private readonly IGetInformationForProject _getInformationForProject;
+        
+        public GeneralInformationViewModel ViewModel { get; private set; }
+        
+        public Index(IGetInformationForProject getInformationForProject)
+        {
+            _getInformationForProject = getInformationForProject;
+        }
+        
+        public async Task<IActionResult> OnGetAsync(string urn)
+        {
+            var getInformationForProjectResponse = await _getInformationForProject.Execute(urn);
+
+            if (!getInformationForProjectResponse.IsValid)
+            {
+                return this.View("ErrorPage", getInformationForProjectResponse.ResponseError.ErrorMessage);
+            }
+            
+            var generalInformation = getInformationForProjectResponse.OutgoingAcademy.GeneralInformation;
+
+            ViewModel = new GeneralInformationViewModel
```

Comment:
View Model isn't needed if no Post method. 
Properties can be moved to the Page model